File: server/paper_config.py
"""Paper configuration shared between client and server."""
import json
import os

def load_paper_config():
    """Load paper configuration from the config JSON file."""
    config_path = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'config', 'papers.json')
    with open(config_path) as f:
        config = json.load(f)
        return config['papers'][config['default']]

# Current paper configuration
CURRENT_PAPER = load_paper_config()

File: client/js/paperConfig.js
export async function loadPaperConfig() {
    try {
        const response = await fetch(`/config/papers.json?v=${Date.now()}`);
        const config = await response.json();
return {
            papers: config.papers,
            default: config.default
        };
    } catch (error) {
        console.error('Error loading paper config:', error);
        return null;
    }
}
